src/google-maps/map-ground-overlay/map-ground-overlay.ts

```diff
@@ -77,21 +86,30 @@ export class MapGroundOverlay implements OnInit, OnDestroy {
   constructor(private readonly _map: GoogleMap, private readonly _ngZone: NgZone) {}
 
   ngOnInit() {
-    if (!this.bounds && (typeof ngDevMode === 'undefined' || ngDevMode)) {
-      throw Error('Image bounds are required');
```

Comment:
Why was the error removed?

Reply:
I removed it since none of the other components that accept bounds had a hard error like this.

Reply:
I think only rectangle accepts a bounds other than this one. The reason that I implemented an error here was that the Google Maps JavaScript API marks bounds as a required input, whereas for rectangle it's optional.

Reply:
Since Angular templates are dynamic, the value could come in asynchronously after `ngOnInit`. I'd rather avoid throwing an error since it isn't any more difficult for us to support it.

Reply:
Okay, I guess that's fine with me.